facdb/data/lookup_classification.csv

```diff
@@ -78,4 +79,4 @@ City Government Offices,"Offices, Training, and Testing",Administration of Gover
 Training and Testing,"Offices, Training, and Testing",Administration of Government,Regional
 Miscellaneous Use,Other Property,Administration of Government,Regional
 Properties Leased or Licensed to Non-public Entities,Other Property,Administration of Government,Regional
-Head Start,Day Care and Pre-Kindergarten,"Education, Child Welfare, and Youth",Local
\ No newline at end of file
+Head Start,Day Care and Pre-Kindergarten,"Education, Child Welfare, and Youth",Local
```

Comment:
I think this change might be breaking things. I thought pre-commit would catch this if working in the dev container.